components/rhel/README.md

```diff
@@ -17,9 +17,9 @@ An additional document, the Container Development Guide, will be published with
 
 Updated Releases
 ================
-Updated releases of the CDK will be available to download from the Red Hat Customer portal, [access.redhat.com](https://access.redhat.com/) under Red Hat Enterprise Linux. Select [Red Hat Container Development Kit](https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software) from the list of product variants.
+Updated releases of the CDK will be available to download from Red Hat Developer portal, [http://developers.redhat.com](http://developers.redhat.com) under Technologies. Select [Red Hat Container Development Kit](http://developers.redhat.com/products/cdk/download/) from the list of product variants.
 
 
-Beta Feedback and Questions
-===========================
-Please join the [container-tools@redhat.com](container-tools@redhat.com) mailing list to ask questions and provide feedback.
+Feedback and Questions
+======================
+Please join the [devtools@redhat.com](devtools@redhat.com) mailing list to ask questions and provide feedback.
```

Comment:
At this point of time for CDK container-tools@redhat.com is still the mailing list. Check the last line in http://developers.redhat.com/products/cdk/overview/